src/pywink/test/devices/air_conditioner_test.py

```diff
@@ -0,0 +1,86 @@
+import json
+import os
+import unittest
+
+import mock
+
+from pywink.api import get_devices_from_response_dict, WinkApiInterface
+from pywink.devices import types as device_types
+from pywink.devices.air_conditioner import WinkAirConditioner
+
+
+class FanTests(unittest.TestCase):
+
+    def setUp(self):
+        super(FanTests, self).setUp()
+        self.api_interface = mock.MagicMock()
+
+    def test_ac_state(self):
+        device_list = []
+        response_dict = {}
+        _json_file = open('{}/api_responses/quirky_aros.json'.format(os.path.dirname(__file__)))
```

Comment:
if you use `with open('...') as _json_file` you don't need to worry about closing the file.